cookbooks/tile/templates/default/export.erb

```diff
@@ -13,6 +13,7 @@ import shutil
 import signal
 import sys
 import tempfile
+from PIL import Image
```

Comment:
Do we need to change the recipe to ensure `PIL` is installed?

Reply:
Probably yes, but I can't tell for sure if PIL hasn't been pulled in by some other dependency already.

Reply:
Well if it's required we should add it to the install list rather than relying on something else pulling it in accidentally - even if that works today it might not in the future.

Reply:
If https://github.com/openstreetmap/chef/blob/master/cookbooks/tile/recipes/default.rb#L102-L105 is the right place, I could of course add it there.

Reply:
https://github.com/openstreetmap/chef/blob/24b4995f48385560ec2a51672f1113f02dadd4cd/cookbooks/tile/recipes/default.rb#L159-L165 is probably better.

Reply:
Added now